src/bridge/events.rs

```diff
@@ -200,6 +200,7 @@ pub enum RedrawEvent {
         anchor_row: f64,
         anchor_column: f64,
         focusable: bool,
+        z_index: i64,
```

Comment:
@Kethku z_index is integer, so i'm using i64 here. But, anchor_row and column also integers, but they using f64.

Should i also change this to f64? Why are they using f64?

Reply:
These values are floats in the spec.
This is so the position of floating windows does not need to be exactly aligned with the character grid in GUIs.

Reply:
@seandewar thats correct. We made that change not that long ago